### PR TITLE
Default module

### PR DESCRIPTION
**Pull request by madushan1000**

### CODE REVIEW

File: lib/modules/default/index.js [hunk 1]
```diff
@@ -26,29 +31,59 @@ export function init(/* api */) {
 
 export function logs(api) {
   log('exec => mup logs');
+  return Promise.all([
+    meteor.logs(api),
+    mongo.logs(api)
```
Comment (arunoda) [inline lib/modules/default/index.js]: here we don't need mongo logs.

File: lib/modules/default/index.js [hunk 2]
```diff
@@ -26,29 +31,59 @@ export function init(/* api */) {
...
+      Promise.all([
+        meteor.setup(api),
+        mongo.setup(api),
+      ]);
+    });
 }
 
 export function start(api) {
   log('exec => mup start');
+  return mongo.start(api)
```
Comment (arunoda) [inline lib/modules/default/index.js]: Do we need to start mongodb? I don't think so.

File: lib/modules/default/index.js [hunk 3]
```diff
@@ -26,29 +31,59 @@ export function init(/* api */) {
...
 }
 
 export function stop(api) {
   log('exec => mup stop');
+  return Promise.all([
+    meteor.stop(api),
+    mongo.stop(api)
+  ]);
 }
+
```
Comment (arunoda) [inline lib/modules/default/index.js]: remove empty spaces.

File: lib/modules/default/index.js (hunk 2 above)
Comment (arunoda) [inline lib/modules/default/index.js]: do this in the setup area.

**Comment by madushan1000** [comment]
added tests

**Pull request merged**